src/Exception/NoCoverageDriverAvailableException.php

```diff
@@ -1,27 +0,0 @@
-<?php
-
-/**
- * This file is part of the friends-of-phpspec/phpspec-code-coverage.
- *
- * @author  shulard <s.hulard@chstudio.fr>
- * @license MIT
- *
- * For the full copyright and license information, please see the LICENSE file
- * that was distributed with this source code.
- */
-
-declare(strict_types=1);
-
-namespace FriendsOfPhpSpec\PhpSpec\CodeCoverage\Exception;
-
-use RuntimeException;
-
-/**
- * When PHPUnit/CodeCoverage trigger an exception which says that's no driver
- * can be found, we catch it to decorate with our own.
- *
- * @author Stéphane Hulard
- */
-class NoCoverageDriverAvailableException extends RuntimeException
```

Comment:
Why removing this specific exception ?

Reply:
Because we just rethrow the same exception from here: https://github.com/sebastianbergmann/php-code-coverage/blob/9.2.3/src/Exception/NoCodeCoverageDriverAvailableException.php

I think that's unnecessary. Should I add it again?

Reply:
This is a case we know and throwing a custom exception help finding where is the issue point instead requiring to check another external lib code...

I use this kind of technique in some projects and find it helpful because it help :

* knowing the possible errors, I mean the one understood in the code logic and that are « normal »,
* make a difference with the others that are not valid from our library perspective.

But it’s my POV, I totally understand your point too 😄.

@drupol what do you think ?

Reply:
@shulard I added the try-catch again. :+1: